src/OrleansProviders/Streams/Memory/MemoryStreamQueueGrain.cs

```diff
@@ -1,4 +1,4 @@
-﻿
+
```

Comment:
Nit: undo that empty line

Reply:
Misbehaved auto-formatter, sorry about that.